Blockzilla/en.lproj/Localizable.strings

```diff
@@ -65,10 +65,10 @@
 "Autocomplete.addCustomUrlWithPlus" = "+ Add Custom URL";
 
 /* Description for adding and managing custom autocomplete URLs */
-"Autocomplete.customDescriptoin" = "Add and manage custom autocomplete URLs.";
+"Autocomplete.customDescriptoin" = "Enable to have Firefox Focus autocomplete your favorite URLs.";
 
 /* Label for button taking you to your custom Autocomplete URL list */
-"Autocomplete.customLabel" = "Custom URLs";
+"Autocomplete.customLabel" = "Manage Sites";
```

Comment:
For all of the string changes you made in this file, you're actually going to want to take make new strings in UIConstants.swift. Unfortunately our localization process does not currently allow for old strings to be updated, so we need to make new NSLocalizableStrings which we'll then use instead of ```autocompleteCustomSectionLabel``` and  ```autocompleteCustomDescription```

Reply:
Cool, I didn't know that! I'll update my commit today or tomorrow. Thanks!